tests/cecil-tests/Test.cs

```diff
@@ -86,5 +87,21 @@ public void EnsureUIThreadOnInit (string assemblyPath)
 					return; // single case, no point in iterating anymore
 			}
 		}
+
+		[TestCaseSource (typeof (Helper), "PlatformAssemblies")]
+		public void NoSystemConsoleReference (string assemblyPath)
+		{
+			if (Path.GetFileName (assemblyPath) == "Xamarin.Mac.dll")
+				Assert.Ignore ("Xamarin.Mac has a workaround for Sierra bug w/NSLog");
```

Comment:
Just out of curiosity: where is this workaround?

Reply:
https://github.com/xamarin/xamarin-macios/blob/944ec6dc2fddffd3cc12d4d5937992f0f7dd6e25/src/ObjCRuntime/Runtime.cs#L1560

Reply:
Commit: https://github.com/xamarin/xamarin-macios/commit/66cbc100fe1fcae69b357ec42159977c71f81c63